.vale/styles/RedHat/PascalCamelCase.yml

```diff
@@ -0,0 +1,51 @@
+---
+extends: existence
+scope: sentence
+ignorecase: false
+nonword: true
+level: warning
+link: https://redhat-documentation.github.io/asciidoc-markup-conventions
+message: "Consider wrapping this Pascal case or Camel case term ('%s') in backticks."
+source: https://github.com/redhat-documentation/vale-at-red-hat/tree/main/.vale/styles/RedHat/PascalCamelCase.yml
+tokens:
+  - ([A-Z]+[a-zA-Z]+){2,}
+  - ([a-zA-Z]+([A-Z][a-z]+))+
+  - Alertmanager
+  - Authentication
+  - Binding
+  - Build
+  - Config
+  - Console
+  - Deployment
+  - Endpoints
+  - Etcd
+  - Event
+  - Group
+  - Help
+  - Identity
+  - Image
+  - Infrastructure
+  - Ingress
+  - Lease
+  - Namespace
+  - Network
+  - Node
+  - OAuth
+  - Pod
+  - Probe
+  - Profile
+  - Project
+  - Prometheus
+  - Provisioning
+  - Proxy
+  - Role
+  - Route
+  - Scheduler
+  - Secret
+  - Storage
```

Comment:
This would trigger false positives for any mentions of Red Hat Ceph Storage, Red Hat Gluster Storage, Red Hat Enterprise Linux Resilient Storage, and similar.

Reply:
@jhradilek Thanks for digging in! I figured this would be tricky for these single word terms, which is why I am hesitant to merge the PR as is.

Reply:
Great feedback - I'll rethink this one ;)